### ISSUE TITLE
Programatically see if screen is in mobile mode

### ISSUE THREAD

**Issue by americanslon** [DerekChasse/Flexor#17]
I don't suppose there is a way to use IsMobile etc. from "code behind" to find out the current viewfinder mode?

**Comment by DerekChasse** [comment]
Flexor isn't intended to programatically access media widths in that fashion.  The IsMobile etc methods are more declarative in nature.

After a quick search looks like Ed Charbeneau has a library which could provide the functionality you're after.

https://github.com/EdCharbeneau/BlazorSize

**Comment by americanslon** [comment]
Yeah I kinda figured. The whole point of me asking was to avoid another library. I just made my own little service.
